mass_mailing_list_dynamic/models/mail_mass_mailing_list.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.tools import safe_eval
+
+
+class MassMailingList(models.Model):
+    _inherit = "mail.mass_mailing.list"
+
+    sync_method = fields.Selection(
```

Comment:
Please add the flag that I specify you: `Dynamic list`. If not marked, the rest of the fields are invisible in the view.

Reply:
It's not needed, just leave this field empty

Reply:
But that's not what I want. Trust me: this is very user unfriendly. Please add the field.